app/javascript/lib/document_submitting_indicator.js

```diff
@@ -0,0 +1,14 @@
+export var documentSubmittingIndicator = (function() {
```

Comment:
There was some code here that was duplicating the implementation of the file uploader code that comes from honeycrisp, so I pulled out the stuff that was duplicated and let this function do what was _unique_ and not supported by the honeycrisp.js implemention, which was adding the submitting indicator properly.

Reply:
Neat! Perhaps we can upstream it someday. :)